tdvt/tdvt/tabquery.py

```diff
@@ -51,6 +51,15 @@ def build_tabquery_command_line(work):
     cmdline = tb.build_tabquery_command_line(work)
     return cmdline
 
+def build_connectors_test_tabquery_command_line(conn_test_name, conn_test_file_name, conn_test_password_file):
```

Comment:
This function doesn't look for the env var at line 14. Can we use `configure_tabquery_path` & `build_tabquery_command_line` here?